apps/storefront/src/utils/validateProducts.ts

```diff
@@ -58,7 +82,7 @@ export const validateProducts = async (products: CustomFieldItems[]): Promise<Va
   settledResults.forEach((result, index) => {
     // Network or unexpected error
     if (result.status === 'rejected') {
-      const { productName } = products[index].node;
+      const { productName } = products[index];
```

Comment:
This should be `productsList` shouldn't it? `products` is the original data with the `node` property

```suggestion
      const { productName } = productsList[index];
```